### ISSUE TITLE
can't launch Safe Examen browser for VMWare

### ISSUE THREAD

**Issue by MagicTrio** [SafeExamBrowser/seb-win-refactoring#1050]
Since few weeks, I can't launch SEB from a virtual machine (VMWARE)

In windows events log i've got
![image](https://github.com/user-attachments/assets/fb773641-2e3a-4561-a725-51821276e010)

I reinstalled SEB with the same result.

Seb reset utility and configure tool are ok... I can launch them.

install version : SEB_3.8.0.742_SetupBundle.exe
os : w11 23h2

I install this version on my laptop (w11 too) and I have no problem.. it's seemed 
Problem seems to come from VMware .. Any idea ?

**Comment by dbuechel** [comment]
Please specify your concrete use case. According to my (limited) experience, SEB works just fine in a Windows 11 VM running on VMware.

**Comment by MagicTrio** [comment]
hi Damien, 

all our vm are in W11 with 23h2 version.
which informations do you need ?

**Comment by dbuechel** [comment]
Have you tried to reinstall / recreate your virtual machine?

**Comment by MagicTrio** [comment]
yes, I recreate.. and same results.. I made tests on old vm ( with same OS 23h2 ) on esxi 6.7(VM vresion15) and it's work...
it doesn't work on esxi 7.0 (VM version 19). I'll ask to vmware admin to move my vm fonrm esxi 6.7 to 7.0 to be sure probleme come on ths version.

**Comment by MagicTrio** [comment]
not with 3.8  but it works with 3.7.1 ...  it better workaround i find.
